openlibrary/templates/work_search.html

```diff
@@ -43,7 +43,7 @@
 $ start_facet_count = 5
 $ facet_inc = 10
 <script type="text/javascript">
-<!--
```

Comment:
Is the commenting removed for a reason?  Was there a commenting close that also needs to be deleted?

Reply:
https://github.com/internetarchive/openlibrary/pull/3977#discussion_r515615618
same issue